### PR TITLE
fix(codebuild): Rename codebuild vpc policy to a unique name

### PR DESCRIPTION
**Pull request by eastNine**
This fix prevents error when a stack has several codebuild projects.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-codebuild/lib/project.ts [hunk 1]
```diff
@@ -955,7 +955,7 @@ export class Project extends ProjectBase {
     }));
 
     const policy = new iam.Policy(this, 'PolicyDocument', {
-      policyName: 'CodeBuildEC2Policy',
+      policyName: this.node.uniqueId + ' CodeBuildEC2Policy',
```
Comment (joehillen) [inline packages/@aws-cdk/aws-codebuild/lib/project.ts]: Are you sure you want a space here?

Comment (eastNine) [reply packages/@aws-cdk/aws-codebuild/lib/project.ts]: It's okay if there's no space.

Comment (joehillen) [reply packages/@aws-cdk/aws-codebuild/lib/project.ts]: No. You misunderstood me. There is a space there that should not be. I'm don't think you can have spaces in policy names.

Comment (eastNine) [reply packages/@aws-cdk/aws-codebuild/lib/project.ts]: Yes,
I understood you mean that space is in policy name.
Should I modify it like this? :
`this.node.uniqueId + 'CodeBuildEC2Policy'`

Comment (joehillen) [reply packages/@aws-cdk/aws-codebuild/lib/project.ts]: Yes

File: packages/@aws-cdk/aws-codebuild/lib/project.ts [hunk 2]
```diff
@@ -955,7 +955,7 @@ export class Project extends ProjectBase {
     }));
 
     const policy = new iam.Policy(this, 'PolicyDocument', {
-      policyName: 'CodeBuildEC2Policy',
+      policyName: this.node.uniqueId + 'CodeBuildEC2Policy',
```
Comment (skinny85) [inline packages/@aws-cdk/aws-codebuild/lib/project.ts]: Can you just remove `policyName` property from this line? This way, the CDK will use the generated ID of the `Policy` construct as the `policyName`.

Comment (eastNine) [reply packages/@aws-cdk/aws-codebuild/lib/project.ts]: Of course, your suggestion seems a better way!

**Pull request merged**